collect_app/src/main/java/org/odk/collect/android/views/MediaLayout.java

```diff
@@ -61,7 +62,7 @@
     private FormIndex mIndex;
     private TextView mView_Text;
     private AudioButton mAudioButton;
-    private ImageButton mVideoButton;
```

Comment:
I bet there is unused import android.widget.ImageButton as you implemented this change :) Please remove that if I'm right.